### PR TITLE
Nav branch - Implemented Firebase Analytics to log user feedback

### PR DESCRIPTION
**Pull request by paulnabanita1**
Please make sure these boxes are checked before submitting your pull request - thanks!

- [ ] Apply the `AndroidStyle.xml` style template to your code in Android Studio.

- [ ] Run the unit tests with `gradlew connectedObaGoogleDebugAndroidTest connectedObaAmazonDebugAndroidTest` to make sure you didn't break anything

- [ ] If you have multiple commits please combine them into one commit by squashing them for the initial submission of the pull request.  When addressing comments on a pull request, please push a new commit per comment when possible (reviewers will squash and merge using GitHub merge tool)

### CODE REVIEW

File: onebusaway-android/src/main/java/org/onebusaway/android/nav/NavigationUploadWorker.java [hunk 1]
```diff
@@ -61,11 +87,31 @@ public void onFailure(@NonNull Exception exception) {
                     @Override
                     public void onSuccess(UploadTask.TaskSnapshot taskSnapshot) {
                         Log.d(TAG, logFileName + " uploaded successful");
+                        String userResponse = taskSnapshot.getMetadata().getCustomMetadata("Response");
```
Comment (barbeau) [inline onebusaway-android/src/main/java/org/onebusaway/android/nav/NavigationUploadWorker.java]: Could you please put these strings in `donottranslate.xml`?

File: onebusaway-android/src/main/java/org/onebusaway/android/nav/NavigationUploadWorker.java [hunk 2]
```diff
@@ -61,11 +87,31 @@ public void onFailure(@NonNull Exception exception) {
...
                         Log.v(TAG, logFileName + " deleted : " + deleted);
                     }
                 });
             }
         }
     }
+
+    private void logFeedback(String feedbackText, String userResponse, String fileName) {
+        Boolean wasGoodReminder;
+        if (userResponse.equals("yes")) {
```
Comment (barbeau) [inline onebusaway-android/src/main/java/org/onebusaway/android/nav/NavigationUploadWorker.java]: Could you please change this hard-coded string to a constant, likely in `donottranslate.xml`?

File: onebusaway-android/src/main/res/values/strings.xml [hunk 3]
```diff
@@ -1106,7 +1106,7 @@
     <string name="es_msg_general_account_deleted">Embedded Social account deleted</string>
 
     <!-- User Feedback notifications -->
-    <string name="feedback_notify_title">Help Improve Destination Reminder</string>
+    <string name="feedback_notify_title">Help improve destination reminders</string>
```
Comment (paulnabanita1) [inline onebusaway-android/src/main/res/values/strings.xml]: Changed title to lowercase.

File: onebusaway-android/src/main/java/org/onebusaway/android/nav/NavigationUploadWorker.java (hunk 1 above)
Comment (paulnabanita1) [reply onebusaway-android/src/main/java/org/onebusaway/android/nav/NavigationUploadWorker.java]: @barbeau Thank you for reviewing the PR. I have added the strings to `donottranslate.xml`.

File: onebusaway-android/src/main/java/org/onebusaway/android/nav/NavigationUploadWorker.java (hunk 2 above)
Comment (paulnabanita1) [reply onebusaway-android/src/main/java/org/onebusaway/android/nav/NavigationUploadWorker.java]: Added hard-coded string to a constant in `donottranslate.xml`.

**Comment by barbeau** [comment]
Thanks @paulnabanita1, looks good!

**Pull request merged**